tests/fixtures/colors/basic.hrx

```diff
@@ -0,0 +1,9 @@
+<===> input.scss
+p {
+  color: #ff8000;
+}
+<===> output.css
+input: ["--precision", "10", "-t", "expanded", "-I", "/Users/alexstephen/sass-spec/spec", "input.scss"]
```

Comment:
We should probably set this up so that that it doesn't require committing machine-specific paths.

Reply:
Oh yes, that's a great point!

Reply:
Changes made!